src/utils/daos_autotest.c

```diff
@@ -113,10 +120,35 @@ step_init(void)
 }
 
 static int
-init(void)
+attachinfo(void)
 {
+	char *command = "sudo daos_agent dump-attachinfo";
+ 	FILE *fp = popen(command, "r");
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs